cosmos/profiles/athena/access_key.py

```diff
@@ -0,0 +1,67 @@
+"Maps Airflow AWS connections to a dbt Athena profile using an access key id and secret access key."
+from __future__ import annotations
+
+from typing import Any
+
+from ..base import BaseProfileMapping
+
+
+class AthenaAccessKeyProfileMapping(BaseProfileMapping):
+    """
+    Maps Airflow AWS connections to a dbt Athena profile using an access key id and secret access key.
+
+    https://docs.getdbt.com/docs/core/connect-data-platform/athena-setup
+    https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/connections/aws.html
+    """
+
```

Comment:
Please, could you add:
 ```suggestion
    is_community: bool = True
```
Similar to:
https://github.com/astronomer/astronomer-cosmos/blob/main/cosmos/profiles/spark/thrift.py